tests/html_glimmer/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -426,6 +426,14 @@ exports[`else-if.hbs 1`] = `
 {{/if}}
 `;
 
+exports[`escaped-chars.hbs 1`] = `
+<p>Some escaped characters: &lt; &gt; &amp; &nbsp; &#160; &#xa0;</p>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<p>
+  Some escaped characters: &lt; &gt; &amp; &nbsp; &nbsp; &nbsp;
```

Comment:
So all of those are turned into `&nbsp;`? Not sure if that’s a feature or a bug. Perhaps it’s out of scope for this PR.